NYC_Summit18_Developer_Workshop/workspace/.metadata/.plugins/org.eclipse.rse.core/profiles/PRF.ip-172-31-29-133_0/H.local_16/node.properties

```diff
@@ -16,7 +16,7 @@
 06-child.00000.06-child.00000.01-type=SubSystem
 06-child.00000.06-child.00000.03-attr.hidden=false
 06-child.00000.06-child.00000.03-attr.type=local.files
-06-child.00000.06-child.00000.06-child.00000.00-name=ip-172-31-90-94___ip-172-31-90-94\:local.files
+06-child.00000.06-child.00000.06-child.00000.00-name=ip-172-31-29-133___ip-172-31-29-133\:local.files
```

Comment:
Same goes here. We should make sure these hostnames are not static. Why keep these?